code/datums/status_effects/neutral.dm

```diff
@@ -342,3 +342,188 @@
 	tick_interval = INFINITY
 	status_type = STATUS_EFFECT_REFRESH
 	alert_type = null
+
+/datum/status_effect/eigenstasium
+	id = "eigenstasium"
+	status_type = STATUS_EFFECT_UNIQUE
+	alert_type = null
+	///So we know what cycle we're in during the status
+	var/current_cycle = -250 //Consider it your stability
+	///The addiction looper for addiction stage 3
+	var/phase_3_cycle = -0 //start off delayed
+	///Your clone from another reality
+	var/mob/living/carbon/alt_clone = null
+	///If we display the stabilised message or not
+	var/stable_message = FALSE
+
+///Convert this effect to process slowly instead
+/datum/status_effect/eigenstasium/on_creation(mob/living/new_owner, ...)
+	. = ..()
+	STOP_PROCESSING(SSfastprocess, src)
+	START_PROCESSING(SSprocessing, src) //this lasts a while, so SSfastprocess isn't needed.
```

Comment:
This should be a refactor at the base of the status effect instead.
Check `processing_flags` for machinery for an example of how to.

Reply:
I'll give it a shot!

Reply:
How's that? I retrofitted it to the other status effects too.